department-of-veterans-affairs/va.gov-cms
# VACMS-9920: Update the main CMS toolbar to meet accessibility standards

## Description

Closes #9920  

## Testing done
Manual visual testing on local machine and tugboat

## Screenshots
![CleanShot 2022-08-22 at 08 14 22](https://user-images.githubusercontent.com/109987005/185957091-30511cfe-ce9d-494c-b0e1-04a5047ebd29.png)
![CleanShot 2022-08-22 at 08 17 50](https://user-images.githubusercontent.com/109987005/185957212-fb279347-94e0-4b48-a74f-470be27fe5de.png)



## QA steps

This will need to be checked on all environments to make sure the colors are correct and that the environment lable is displayed correctly.  

Tugboat link is: https://pr10365-1zz2vdelkvfvgeighw1xgkqhc5xlilwr.ci.cms.va.gov/

As super admin
1. Log in
   - [x] Validate that commit hash, branch, or build version appear and environment label is present.

As editor
1. Log in
   - [x] Validate that commit hash, branch, or build version DO NOT appear and environment label is present.

### Definition of Done


### Select Team for PR review


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
